projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -902,6 +903,8 @@ async function write(options, preview, contents) {
 	else {
 		await writeFileAsync(options.outfile, contents);
 	}
+
+	yarn('format');
```

Comment:
Note that this line runs unconditionally, bypassing the `--dry-run` flag. Not a huge deal, but if a tool mutates that filesystem when running with `--dry-run`, user will probably interpret it as a bug.

Reply:
Good catch :+1:  (who's this guy, BTW? :thinking: )

Reply:
Fixed!! My fault, I didn't realise that when `--dry-run` option is executed we do not write the file, but just show a preview. 
Thanks @wincent and @izaera!